src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>UkhoFileshareserviceUi</title>
+  <title>FSS-UI</title>
   <base href="/">
```

Comment:
I can see this has already been updated but i don't think either of the above options are useful titles for this page. An end user is unlikely to understand an abbreviation.

I would have thought the page title should be "UK Hydrographic Office File Share Service" which might be meaningful to an end user.

Was the title specified somewhere?

Reply:
We have discussed this issue and have agreed to accept this code as it is currently, merge into main and review any required changes in a subsequent PBI.